products/node-feature-discovery.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Kubernetes Node Feature Discovery
+category: server-app
+tags: kubernetes
+iconSlug: kubernetes
+permalink: /kubernetes-node-feature-discovery
+alternate_urls:
+-   /k8s-node-feature-discovery
+-   /node-feature-discovery
+versionCommand: |-
+  # Must be installed through helm chart https://artifacthub.io/packages/helm/node-feature-discovery/node-feature-discovery
+  nfd-gc -version
+  nfd-master -version
+  nfd-topology-updater -version
+  nfd-worker -version
```

Comment:
Are these likely to report different versions? I'd prefer if we could provide a single command.

Alternatively, since the version is same as on the docker image, would a kubectl command be better here ([list the image across all namespaces (with a grep)](https://kubernetes.io/docs/tasks/access-application-cluster/list-all-running-container-images/#list-all-container-images-in-all-namespaces)). That would work on non-helm installs as well.

Reply:
These all report the same version - but I'll be swapping it with the kubectl list images command instead.